MCE 2.2.10

dockerymick opened the pull request:
Related PR for ACM 2.7.10: https://github.com/stolostron/rhacm-docs/pull/5925/files
Asked release manager about doc issue for this. I need help with confirming which bugs need to be mentioned in the Errata

Query with issues that do not have the `errata` label, but have Errata version in the Fix versions

dockerymick committed:
MCE 2.2.10

cadawson commented:
It looks like there are no issues to be listed in the errata for MCE this round. I think the one that you pulled in was still open. It will probably have to move ti the next release. I think we go with the standard container updates for this one, I guess. Thanks!

cadawson reviewed (approved):
Looks good. Might be that we only go with the standard updates here and remove the comments. Thanks! I will also add Robyn to our GH group. I don't think she is already there. Thanks!

oafischer reviewed (approved):
Looks good if no issues are intended to be tagged.

dockerymick commented:
Here is the query: https://issues.redhat.com/browse/ACM-8844?jql=project%20%3D%20ACM%20AND%20fixVersion%20in%20(%22MCE%202.2.10%22)%20

cadawson commented:
Hey @dockerymick. Just a heads-up that that link is to an issue. It is not the query. This is a link to the query: https://issues.redhat.com/issues/?filter=12425893. I gave you and @rhaignere access to it. Let me know if it doesn't work. Thanks!

dockerymick commented:
@cadawson the query that you shared is for ACM 2.7.10. 
<img width="1726" alt="Screenshot 2023-12-11 at 9 18 34 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/b5f5091b-ee9a-489a-8e46-9af26e17ad28">

Why is Ordered by ASC an important label? Does that mean Ascending? 

When I edited the query that you shared to only have MCE, I received no results:
<img width="1515" alt="Screenshot 2023-12-11 at 9 22 15 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/24c4440f-d9ac-4cdf-bcee-5f78975045eb">

When i removed ACS:

<img width="1520" alt="Screenshot 2023-12-11 at 9 23 27 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/cb930629-ec72-4828-9b66-b8fdcb456978">

Results after removing errata label:

<img width="1527" alt="Screenshot 2023-12-11 at 9 24 30 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/0561a4f0-642b-49e5-a098-59837b7f9504">


Results when I added labels ASC, no errata label:

<img width="1529" alt="Screenshot 2023-12-11 at 9 25 59 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/f26b72a3-8477-49f0-ba39-17aed2472fda">

cadawson commented:
@dockerymick: I got you. The view that it pulled up for your query only showed the single issue. It still does, but I see that I can scroll through the other issues with the arrows in the upper right corner. I did notice the you did not include the errata label. That is how it listed MCE 2.2.10 issues for you. 

Here is the confusion: My query includes both MCE 2.2.10 and ACM 2.7.10, but only the issues that contain the "errata" label. You can separate them into separate MCE and ACM searches, if it helps. Those are the only ones that the docs team is concerned with. Those are the ones that @rhaignere wants us to include because she added the "errata" label to it. In this case, it does not look like there are any MCE 2.2.10 issues for us to include. That might be incorrect until we get the process working, but that fix version and the "errata" label are what we filter on. If it is not there, then we assume that we don't want to expose any of the bug fixes publicly, or there were not any. In that case, we include the standard container updates notification and don't list any bug fixes in the docs errata. 

It might be that you were pulling all of the 2.2.10 issues to help Robyn see which ones were candidates for the errata label. In that case, your query works for what you are looking for. That was cool of you to do that. My query was more selfish in that it is what the doc team needs for the errata.  

You are correct that ASC is ascending order for the issues that match the filters. It should not affect the filtering aside from ordering them in the window. 

Does that help?

dockerymick commented:
@cadawson thanks for reminding me of the process and confirming the query. You are correct about my intentions lol thanks!

dockerymick commented on clusters/release_notes/errata.adoc:
```diff
@@ -7,6 +7,14 @@ For {mce-short}, the Errata updates are automatically applied when released.
+
+* Delivers updates to one or more of the product container images.
+
+//need help listing the bug fixes for MCE 2.2.10
+
+//https://issues.redhat.com/browse/ACM-8844?jql=project%20%3D%20ACM%20AND%20fixVersion%20in%20(%22MCE%202.2.10%22)%20
```
Removing hidden comment
```suggestion
```

dockerymick committed:
Update clusters/release_notes/errata.adoc

dockerymick commented:
Merging this PR
